### PR TITLE
feat(ext2020): delete ext2020

### PR DESCRIPTION
**Pull request by kaka-lin**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [ ] **New feature**
- [x] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**

Remove `ext2020`

## Related Issue

 [Revamp `ext2020` app](https://trello.com/c/vYNToMLT/50-revamp-ext2020-app)

### CODE REVIEW

File: src/ext2020/README.md [hunk 1]
```diff
@@ -1,37 +0,0 @@
-## Live broadcast settings
```
Comment (mattwang44) [inline src/ext2020/README.md:1]: Please modify this readme and move to `attendee` app

**Comment by mattwang44** [comment]
Just list some items that need to be done in this PR
- Determine how to delete the table from db
- Update i18n files (run `python manage.py makemessages`)

File: src/ext2020/README.md (hunk 1 above)
Comment (kaka-lin) [reply src/ext2020/README.md:1]: OK

**Review by setmao — CHANGES_REQUESTED** [review]

File: src/attendee/README.md [hunk 2]
```diff
@@ -10,19 +13,20 @@ To do so, navigate to `/admin/registry/entry/` and create or update the key.
...
 The youtube video id is the `v` query parameter in the URL of a youtube video, something that looks like `Z_K9Bwgjiuc`.
 
-### Test
+### For Testing
 
 In order to test it out, you will need to manual add a test attendee token.
 The token needs to be a 32 byte hex string (`[0-9a-f]{32}`).
 
-Go to `/admin/ext2020/attendee/` and add an entry.
+Go to `/admin/attendee/attendee` and add an entry.
```
[the remark this answers is not in the document]
Comment (mattwang44) [reply src/attendee/README.md:21]: not a typo. The 1st one is app name and the 2nd one is model name.

File: src/attendee/README.md [hunk 3]
```diff
@@ -10,19 +13,20 @@ To do so, navigate to `/admin/registry/entry/` and create or update the key.
...
 Put `1234567890abcdef1234567890abcdef` in the token field for instance,
 and make sure to make `verified` field `True`.
 
 After creating the attendee entry, you can see the live braodcast page at `/ext/live?token=1234567890abcdef1234567890abcdef`.
 
 
 ## Attendee Token
+
 We can import valid tokens by using the import / export tool in admin.
 Go to `/admin/ext2020/attendee/` and click the import button on the top-right corner.
```
[the remark this answers is not in the document]
Comment (kaka-lin) [reply src/attendee/README.md]: updated it!

File: src/postgres/migrations/0009_delete_ext2020.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    ]
+
+    operations = [
+        migrations.RunSQL('''
+        drop table if exists ext2020;
+        delete from auth_permission where content_type_id in (select id from django_content_type where app_label = '{app_label}');
+        delete from django_admin_log where content_type_id in (select id from django_content_type where app_label = '{app_label}');
+        delete from django_content_type where app_label = '{app_label}';
+        delete from django_migrations where app='{app_label}';
+        '''.format(app_label='ext2020'))
```
Comment (setmao) [inline src/postgres/migrations/0009_delete_ext2020.py:18]: I think using `./manage.py migrate ext2020 zero` before this PR merged is better than use SQL to delete the table.
ref: https://stackoverflow.com/questions/3329773/django-how-to-completely-uninstall-a-django-app

**Review by setmao — CHANGES_REQUESTED** [review]

File: src/postgres/migrations/0009_delete_ext2020.py (hunk 4 above)
Comment (kaka-lin) [reply src/postgres/migrations/0009_delete_ext2020.py:18]: Hi @setmao 

I  have used this command in my local environment. 
But I don't how to do this command in product environment

Comment (mattwang44) [reply src/postgres/migrations/0009_delete_ext2020.py:18]: I'll handle the deployment so don't worry about staging/production.

Comment (setmao) [reply src/postgres/migrations/0009_delete_ext2020.py:18]: @kaka-lin I think we can delete this file and let @mattwang44 handle the migration in staging/production.

Comment (kaka-lin) [reply src/postgres/migrations/0009_delete_ext2020.py:18]: @setmao OK

**Review by setmao — APPROVED** [review]
LTGM

**Pull request merged**